packages/application/src/radix.ts

```diff
@@ -457,25 +457,52 @@ const create = (
 			txLog.debug('Starting signing transaction (async).')
 			return activeAccount.pipe(
 				take(1), // IMPORTANT!
+				withLatestFrom(transactionIntent$),
```

Comment:
unsure if this is incorrect @AbstractFruitFactory ? not firing for test `it('can attach messages to transfers and encrypt them'` hmm...

Reply:
i.e. we do not go into  the body below in the mergeMap.

Reply:
fixed with `combineLatest`.